core/src/main/scala/eventstore/core/SystemError.scala

```diff
@@ -9,29 +10,28 @@ trait SystemError extends RuntimeException with NoStackTrace with Serializable
 
 sealed trait ServerError extends SystemError
 
-@SerialVersionUID(1L) case object BadRequest extends ServerError
-
+@SerialVersionUID(1L) case object BadRequest       extends ServerError
 @SerialVersionUID(1L) case object NotAuthenticated extends ServerError
 
 sealed trait OperationError extends ServerError
 
 object OperationError {
-  @SerialVersionUID(1L) case object PrepareTimeout extends OperationError
-  @SerialVersionUID(1L) case object CommitTimeout extends OperationError
-  @SerialVersionUID(1L) case object ForwardTimeout extends OperationError
+  @SerialVersionUID(1L) case object PrepareTimeout       extends OperationError
```

Comment:
what if we will drop all `@SerialVersionUID(1L)` ?

Reply:
Agree